4.analyze_data/notebooks/UMAP/image_settings/failing_drug_x_left.yaml

```diff
@@ -0,0 +1,56 @@
+failing-drug_x-left:
+  UMAP0: [-1, 0]
+  UMAP1: [6.25, 7.5]
+  Random state: 30
```

Comment:
What are each of these random states? I thought each UMAP was generated through one random state seed?

Reply:
In this context, the random state refers to selecting a random row in the UMAP data frame given the UMAP parameters. 

I could also call it "Random Row" but the notebook is using the `random_state` parameter from the `sample` function, so I thought it made sense to keep the names the same.

I will need to make sure this is clearer!